examples/defun.kl

```diff
@@ -1,22 +1,6 @@
 #lang "prelude.kl"
 [import [shift "prelude.kl" 1]]
```

Comment:
we're not defining any macros, so the shifted import is no longer necessary.

Reply:
Nice catch!